docs(readme) : Updated readme with macOS installation for gCloud sdk for #5

yashcrest opened the pull request:
This PR adds macOS installation guide for google-cloud-cli. 

closes #5

yashcrest committed:
docs(readme) : Update readme to section for #5

boltdynamics commented on Readme.md:
````diff
@@ -36,6 +36,22 @@ Xplorers API is the backend service for the Xplorers website and other related t
+    ```
+2. Extract and move (.tar.gz) file into `$home` path.
+    ```
+    tar -xzvf filename.tar.gz -C ~
````
Can you also add a cleanup command at the end to remove the zip and other unnecessary artefacts?

yashcrest committed:
fix: added clean up command to remove gcloud-sdk .zip

boltdynamics commented:
Actually, why is there a change to the `package-lock.json` file?

yashcrest committed:
Delete package-lock.json

I might have mistakenly run `npm i` in my branch instead of `pnpm i` . 

I have deleted it from this commit.

yashcrest commented:
@boltdynamics dai could you approve and merge this? I mistakenly dismissed your review last week.
